files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1150,7 +1150,7 @@ MyApplet.prototype = {
         this.settings.bind("activate-on-hover", "activateOnHover", this._updateActivateOnHover);
         this._updateActivateOnHover();
 
-        this.menu.actor.add_style_class_name('menu-background');
+        this.menu.setCustomStyleClass('menu-background');
```

Comment:
@collinss 
> In fact, we want all the other menus to be uniform in appearance. 

Sorry that doesn't make much sense to me. I mean why even have themes in the first place if you don't allow them to use whatever styles their designers want to use? Consistency is important, all of the official Cinnamon/Linux Mint themes should definitely be consistent. That can easily be achieved while at the same time allowing unofficial theme authors to be as creative as they like.  

The bottom line is that implementing what I described costs the project absolutely nothing thanks to the work you already did on this PR. That extra style class won't change anything for users of themes that don't utilize that class.